_about/notes/0.6.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Istio 0.6
+
+order: 95
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Custom Envoy Config**. Pilot now supports ferrying cuatom Envoy config to the
+proxy. TBD: link?
+
+- **External Services**. TBD: what can we say here?
+
+- **Gateway Implementation**.
+
+## Mixer adapters
+
+- **SolarWinds**. Mixer can now interface to AppOptics and Papertrail.
+
+- **Redisquota**. Mixer now supports a Redis-based adapter for rate limit tracking. TBD: Doc link?
+
+- **Dogstatsd**. Mixer now supports the dogstatsd metric adapter. TBD: doc link
+
+## Other
```

Comment:
need to say something about
https://github.com/istio/istio/pull/3272

and the other:
https://github.com/istio/istio/pulls?q=is%3Apr+is%3Aclosed+label%3A%22kind%2Fimportant+for+release+notes%22